object_tracker/custom_tracker.py

```diff
@@ -3,11 +3,28 @@
 from pytracking.evaluation.multi_object_wrapper import MultiObjectWrapper
 from typing import List, Tuple
 from numpy import ndarray
-
+import numpy as np
 
 class CustomTracker(Tracker):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        params = self.get_parameters()
+
+        debug_ = getattr(params, "debug", 0)
+        params.debug = debug_
+
+        params.tracker_name = self.name
+        params.param_name = self.parameter_name
+
+        self.tracker = self.create_tracker(params)
+        if hasattr(self.tracker, "initialize_features"):
+            self.tracker.initialize_features()
+        image = np.zeros((512, 640, 3), dtype=np.uint8)
+        bb = [10, 10 ,10 ,10]
+        self.init_tracker(image, bb)
```

Comment:
Image dimensions should not be hard-coded but parameterised. 

The default bounding box should have a more indicating name (e.g., dummy_bb), as it is only used for the first initialisation.